terraform-provider-constellation/internal/provider/cluster_resource.go

```diff
@@ -449,8 +449,8 @@ func (r *ClusterResource) ModifyPlan(ctx context.Context, req resource.ModifyPla
 
 	licenseID := plannedState.LicenseID.ValueString()
 	if licenseID == "" {
-		resp.Diagnostics.AddWarning("Constellation license not found.",
-			"Using community license.\nFor details, see https://docs.edgeless.systems/constellation/overview/license")
+		resp.Diagnostics.AddWarning("Constellation license ID not set.",
+			"Will use community license.")
```

Comment:
```suggestion
			"Continuing with community license.")
```
nit: Sounds less anthropomorphized imo